flag_path.go

```diff
@@ -56,7 +56,8 @@ func (f *PathFlag) GetValue() string {
 
 // Apply populates the flag given the flag set and environment
 func (f *PathFlag) Apply(set *flag.FlagSet) error {
-	if val, ok := flagFromEnvOrFile(f.EnvVars, f.FilePath); ok {
+	// TODO: how to report the source of parse errors?
```

Comment:
My question here is—what parse errors?

It seems like the source is always reported as part of an error in the `Appy` function, but if this function doesn't return an error, I'm not sure we're meant to do anything.

Reply:
Removed.